Whiskey/Whiskey.psd1

```diff
@@ -150,6 +150,8 @@
 # 0.35.0
 
 * Created "SetVariableFromPowerShellDataFile" task for creating variables from values in PowerShell data files (e.g. .psd1 files, module manifests, etc.).
+* Fixed: `ProGetUniversalPackage` task should remove the temporary robocopy output file after it's no longer necessary so it doesn't inadvertently get added to package with a "*.txt" wildcard Include item.

```

Comment:
If somebody is including files under .output in their package, we should let them. You're putting the output file in your tasks's temp directory, which should get cleaned up automatically when the task completes. I'm curious how/where you saw this file in a package?

Reply:
My wording on the issue/fix was actually incorrect. What's actually happening is we use the root of the task temp directory as the temp staging area for the package contents. At the end of the task we zip up the entire task temp directory to create the upack package. So since the robocopy output files are being created in the task's temp directory, they're being zipped up with the package next to the upack.json.

That being said, I think the better solution is to move the temp staging area for the package one level deeper in the task's temp directory. Thoughts?

Reply:
Either approach works for me. Your decision.

Reply:
I moved the temp package staging directory one level below the root task temp directory to prevent any sort of misc task files from getting package in the future.